member/models.py

```diff
@@ -133,7 +134,7 @@ def motd(self):
 
     def as_email_recipient(self):
         return EmailRecipient(name=self.username, email=self.email,
-                              language=self.preferences.language)
+                              language=getattr(self,'preferences').language)
```

Comment:
You can turn off a pylint error for a single line by adding a comment: `# pylint: disable=error-name`.  I'd prefer that to this sort of trickery.

Reply:
And it looks like you already knew that.  But I still think it's appropriate here as well.